doc/src/manual/types.md

```diff
@@ -502,6 +502,26 @@ ERROR: setfield!: const field .b of type Baz cannot be changed
 [...]
 ```
 
+It's important to note that the `mutable struct` keyword does not declare a new type that is
+different from the one defined with the `struct` keyword and vice-versa, if both types have
+the same name and the declaration was done in the same namespace.
+
+As an example, declaring a type with name as `Point` using `struct` is also equivalent to the
```

Comment:
I don't understand this sentence at all. How is it equivalent? One is mutable the other is not.

Reply:
Oh that's true. Have a better wording there?

Reply:
I think the new one is better.